sites/default/sqlconf.php

```diff
@@ -1,26 +1,29 @@
 <?php
 //  LibreEHR
-//  MySQL default Config
-//  Referenced from /library/sqlconf.php.
-
-global $disable_utf8_flag;
-$disable_utf8_flag = false;
+//  MySQL Config
 
 $host	= 'localhost';
 $port	= '3306';
 $login	= 'libreehr';
-$pass	= 'libreehr';
+$pass	= 'toor';
 $dbase	= 'libreehr';
 
+global $disable_utf8_flag;
+$disable_utf8_flag = false;
```

Comment:
This is NOT suppose to be in this commit. Please remove this file from this commit

Reply:
Done